__tests__/api.test.js

```diff
@@ -126,12 +126,25 @@ describe.only("5. PATCH /api/reviews/:review_id", () => {
           review_body: "We couldn't find the werewolf!",
           review_img_url:
             "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
-          votes: 68988,
+          votes: 17,
           category: "social deduction",
           owner: "bainesface",
           designer: "Akihisa Okui",
           created_at: "2021-01-18T10:01:41.251Z",
         });
       });
   });
+  it("status:201, responds with the updated review with new votes", () => {
+    const updatedVote = { inc_votes: "sdf" };
+    return request(app)
+      .patch("/api/reviews/3")
+      .send(updatedVote)
+      .expect(400)
+      .then(response => {
+        expect(response.body).toEqual({
+          status: 400,
+          msg: "invalid vote data",
+        });
+      });
+  });
```

Comment:
Theres still some more testing we could do here for error handling such as;
- sending a request body with the incorrect key 'votes' vs 'inc_votes'
- trying to patch a review id that is out of range (1000)
- trying to patch a review id that is not a number

Anything the user has control over, we need to be extra vigilant